types/json2csv/index.d.ts

```diff
@@ -1,18 +1,32 @@
-// Type definitions for json2csv 4.2
+// Type definitions for json2csv 4.4
 // Project: https://github.com/zemirco/json2csv
-// Definitions by: Juanjo Diaz <https://github.com/juanjoDiaz>
+// Definitions by: Daniel Gooß <https://github.com/dangoo>
```

Comment:
Although I appreciate the additions, I'm not convinced that the name of the main maintainer should be completely removed in favour of someone not related to the development of the library 🙂

Reply:
Yeah, you're totally right